Set/SetGame.swift

```diff
@@ -0,0 +1,269 @@
+//
+//  Set.swift
+//  Set
+//
+//  Created by Alon Reik on 15/03/2021.
+//
+
+import Foundation
+
+struct SetGame
+{
+    
+    /* -------
+     Properties
+     -------- */
+    
+    // An arbitrary number utilized to score matches (update score based on matches) based on number of open cards.
+    // (The view controller also uses a timer to decreases this value if the player takes a long time to find a match).
+    var BaseForScore = 240
+    
+    var score = 0
+    var enemyScore = 0
+    private var scoreUpdate: Int {
+        // every time a match is found, the scoring update depends on the
+        // number of open cards. (more open card = less score)
+        get {
+            return BaseForScore / openCards.count // (integer division)
+        }
+    }
+    
+    var gameIsOver: Bool {
+        get {
+            return deck.isEmpty && findMatchInOpenCards() == nil
+        }
+    }
```

Comment:
I saw that you like to use computed variables. In this cases it will be more clear to write:
```
var gameIsOver = {
   return deck.isEmpty && findMatchInOpenCards() == nil
}()
```

Reply:
Also, why it is a var?